sample/Readme.md

```diff
@@ -21,6 +21,8 @@ In a different terminal:
 cargo run ./examples/configurations/bob.yml
 ```
 
+Update the [example contract](./examples/contracts/numerical_contract_input.json#L82) replacing the number after `btcusd` with a unix timestamp sometimes in the future (this will correspond to the contract maturity date).
```

Comment:
```suggestion
Update the [example contract](./examples/contracts/numerical_contract_input.json#L82) replacing the number after `btcusd` with a unix timestamp some time in the future (this will correspond to the contract maturity date).
```

Reply:
Thanks fixed.